fili-core/src/main/java/com/yahoo/bard/webservice/web/FilteredThetaSketchMetricsHelper.java

```diff
@@ -387,7 +387,7 @@ public PostAggregation replacePostAggWithPostAggFromMap(
             Aggregation aggregation,
             DimensionDictionary dimensionDictionary,
             LogicalTable table,
-            DataApiRequest apiRequest
+            DataApiRequestImpl apiRequest
```

Comment:
This change doesn't look right. Shouldn't you be passing the interface here instead of the class? If not, it deserves a specific comment saying why the interface was not sufficient.